tvu_net_cron/models/cancel_expired_quotation.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api 
+from datetime import date
+
+
+class cancelExpiredQuotation(models.Model):
```

Comment:
Follow conventions: https://www.odoo.com/documentation/15.0/contributing/development/coding_guidelines.html#symbols-and-conventions

Also, does this need to be it's own model? Is there a model that this would make sense to extend?